src/batchify.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * @license Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+/**
+ * Creates set of operations which need to be applied to the input in order to transform
+ * it to the output. Can be used with strings or arrays.
+ *
+ *		const input = Array.from( 'abc' );
+ *		const output = Array.from( 'xaby' );
+ *		const batch = batchify( diff( input, output ), output );
+ *
+ *		batch.forEach( operation => {
```

Comment:
I don't like it either. Not only it collides with names we already have, but is not strictly connected with what it does. What I mean is that it could be called any way (i.e. `groupify`, `groupDiff`, `diffOnContext`), so there is no pressure on keeping this name. I am sure we will find a better name :)